src/platforms/javascript/common/session-replay/privacy.mdx

```diff
@@ -51,6 +51,26 @@ Additionally, Sentry attempts to [scrub](/product/data-management-settings/scrub
 
 More details about this feature can be found in the [configuration page](/platforms/javascript/session-replay/configuration/#network-details).
 
+## Custom scrubbing
+
+As of SDK version 7.53.0, `beforeAddRecordingEvent` was added that allows you to scrub and/or filter out recording events before they leave the browser. These events include console logs and network request and response data.
```

Comment:
I'm aiming for something that describes "scrub" a little more. It might be a word we use internally a lot, but other people don't hear it as often and need the hint.

```suggestion
As of SDK version 7.53.0, `beforeAddRecordingEvent` was added that allows you to modify, scrub (to remove PII) or ignore recording events before they leave the browser. These events include console logs and network request and response data.
```